src/base/system_parameter.c

```diff
@@ -2040,6 +2041,10 @@ bool PRM_ER_LOG_READ_DATA_PAGE = true;
 static bool prm_er_log_read_data_page_default = true;
 static unsigned int prm_er_log_read_data_page_flag = 0;
 
+bool PRM_ER_LOG_CALC_REPL_DELAY = true;
+static bool prm_er_log_calc_repl_delay_default = true;
```

Comment:
I should have payed more attention. By default the debug logging parameters should be false.

An issue was open to change the default value for prm_er_log_read_log_page_default and prm_er_log_read_log_page_default too.

Reply:
Done